### PR TITLE
SPORTSHUB-81 Support float pricing

### PR DESCRIPTION
**Pull request by owya490**
now we can have $18.88 as the price for events.

Events are now stored as cents in the database.

### CODE REVIEW

File: frontend/utilities/priceUtils.ts [hunk 1]
```diff
@@ -0,0 +1,12 @@
+export function dollarsToCents(dollars: number): number {
+  return Math.round(dollars * 100);
+}
+
+export function centsToDollars(cents: number): number {
+  return parseFloat((cents / 100).toFixed(2));
+}
+
+export function displayPrice(price: number): number {
```
Comment (brian2w) [inline frontend/utilities/priceUtils.ts:9]: Put a short doc above this function seeing that this is used extensively throughout the codebase.

**Comment by brian2w** [comment]
There are a couple of current TODO's with the following comment
image
Do these need to be changed as well for price?

**Review by brian2w — CHANGES_REQUESTED** [review]

**Review by ReggieStar11 — COMMENTED** [review]
looks good just fix up brians comments

**Review by brian2w — APPROVED** [review]
lgtm

**Review by ReggieStar11 — APPROVED** [review]

**Pull request merged**